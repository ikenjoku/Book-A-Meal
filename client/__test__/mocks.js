
export const meals = [
  {
    id: 0,
    name: 'Nigerian Jollof',
    description: 'Max Jollof rice, plantain, grilled turkey, bellefull Coke',
    price: 1100,
    imageurl: 'https://africa-publicmages/products/80/80418_1465475724_ma.jpg',
    createdAt: new Date(),
    updatedAt: new Date(),
  },
  {
    id: 1,
    name: 'Rice and Beans',
    description: 'Rice, Beans, Plantain, Panla Sauce, Max Coke',
    price: 1500,
    imageurl: 'https://africa-g/images/products/80/80418_1465475724_ma.jpg',
    createdAt: new Date(),
    updatedAt: new Date(),
  },
  {
    id: 2,
    name: 'Egusi Semo',
    description: 'Semolina, Egusi Soup, Meat, Extra',
    price: 1200,
    imageurl: 'https://africa-public.food.jcts/80/80418_1465475724_ma.jpg',
  },
  {
    id: 3,
    name: 'Green Calulu',
    description: 'Chicken salad with extra cabbage',
    price: 1250,
    imageurl: 'https://dynamic/production/ng/images/products/80/_ma.jpg',
    createdAt: new Date(),
    updatedAt: new Date(),
  },
  {
    id: 4,
    name: 'Onion Jappatti',
    description: 'Ingera palate with raw meat sauce',
    price: 2200,
    imageurl: 'https://dynamic/production/ng/images/produc475724_ma.jpg',
    createdAt: new Date(),
    updatedAt: new Date(),
  },
];

export const testMeal = {
  id: 3234,
  name: 'Strawberry flakes',
  description: 'Ingera palate with raw meat sauce',
  price: 2200,
  imageurl: 'https://dynamic/prodg/images/produc475724_ma.jpg',
  createdAt: new Date(),
  updatedAt: new Date(),
};

export const mockStoreData = {
  authReducer: {
    authLoading: true,
    user: {},
    isLoggedIn: false,
    error: null,
  },
  mealReducer: {
    meals: [
      {
        id: 3,
        name: 'Green Calulu',
        description: 'Chicken salad with extra cabbage',
        price: 1250,
        imageurl: 'https://dynamic/production/ng/images/products/80/_ma.jpg',
        createdAt: new Date(),
        updatedAt: new Date(),
      },
      {
        id: 4,
        name: 'Onion Jappatti',
        description: 'Ingera palate with raw meat sauce',
        price: 2200,
        imageurl: 'https://dynamic/production/ng/images/produc475724_ma.jpg',
        createdAt: new Date(),
        updatedAt: new Date(),
      },
    ],
    error: null,
    isLoading: false,

  },
  menuReducer: {
    selectedMenu: null,
    error: null,

  },
  setMenuReducer: {
    createdMenu: null,
    selectedDate: '',
    error: null,
    success: false,
    updatedMenu: null,
  },
  orderReducer: {
    orderedMeal: null,
    error: null,
    meals: [],
    orders: [],
    previousOrders: [],
    isLoadingOrders: false,
  },
};

export const createdMenu = {
  message: 'Menu for 2018-06-15 was successfully created',
  newMenu: {
    id: 24,
    date: '2018-06-15',
    updatedAt: '2018-07-02T17:37:46.023Z',
    createdAt: '2018-07-02T17:37:46.023Z',
  },
};

export const menus = [{
  id: 23,
  date: '2018-07-02',
  createdAt: '2018-07-01T23:56:23.508Z',
  updatedAt: '2018-07-01T23:56:23.508Z',
  Meals: [
    {
      id: 31,
      name: 'Beans and Bread',
      price: 800,
      description: 'Heavily loaded with smoked fish.',
      imageurl: 'https://res.cloudinary.com/ikeenjoku/image/upload/v1529133645/bookameal/2018-06-16T07:20:43.822ZVegetablerice%2Bplantain%2Bgoatmeat%2Bcoke.jpg.jpg',
      MealMenu: {
        mealId: 31,
        menuId: 23,
        createdAt: '2018-07-01T23:56:28.012Z',
        updatedAt: '2018-07-01T23:56:28.012Z',
      },
    },
    {
      id: 34,
      name: 'Fish balls soup',
      price: 1200,
      description: 'Deliciously light soup with white fish balls',
      imageurl: 'https://res.cloudinary.com/ikeenjoku/image/upload/v1529134583/bookameal/2018-06-16T07:36:22.300Zassortedmeat.jpg.jpg',
      MealMenu: {
        mealId: 34,
        menuId: 23,
        createdAt: '2018-07-01T23:56:32.313Z',
        updatedAt: '2018-07-01T23:56:32.313Z',
      },
    },
    {
      id: 30,
      name: 'Beef with veggie soup',
      price: 1150,
      description: 'Stir fried sliced beef with premium vegetables and spices',
      imageurl: 'https://res.cloudinary.com/ikeenjoku/image/upload/v1529136037/bookameal/2018-06-16T08:00:34.302ZUgwu-Archi.jpg.jpg',
      MealMenu: {
        mealId: 30,
        menuId: 23,
        createdAt: '2018-07-01T23:56:36.920Z',
        updatedAt: '2018-07-01T23:56:36.920Z',
      },
    },
  ],
},
{
  id: 24,
  date: new Date().toISOString().substr(0, 10),
  createdAt: '2018-07-01T23:56:23.508Z',
  updatedAt: '2018-07-01T23:56:23.508Z',
  Meals: [
    {
      id: 31,
      name: 'Beans and Bread',
      price: 800,
      description: 'Heavily loaded with smoked fish.',
      imageurl: 'https://res.cloudinary.com/ikeenjoku/image/upload/v1529133645/bookameal/2018-06-16T07:20:43.822ZVegetablerice%2Bplantain%2Bgoatmeat%2Bcoke.jpg.jpg',
      MealMenu: {
        mealId: 31,
        menuId: 23,
        createdAt: '2018-07-01T23:56:28.012Z',
        updatedAt: '2018-07-01T23:56:28.012Z',
      },
    },
    {
      id: 34,
      name: 'Fish balls soup',
      price: 1200,
      description: 'Deliciously light soup with white fish balls',
      imageurl: 'https://res.cloudinary.com/ikeenjoku/image/upload/v1529134583/bookameal/2018-06-16T07:36:22.300Zassortedmeat.jpg.jpg',
      MealMenu: {
        mealId: 34,
        menuId: 23,
        createdAt: '2018-07-01T23:56:32.313Z',
        updatedAt: '2018-07-01T23:56:32.313Z',
      },
    },
    {
      id: 30,
      name: 'Beef with veggie soup',
      price: 1150,
      description: 'Stir fried sliced beef with premium vegetables and spices',
      imageurl: 'https://res.cloudinary.com/ikeenjoku/image/upload/v1529136037/bookameal/2018-06-16T08:00:34.302ZUgwu-Archi.jpg.jpg',
      MealMenu: {
        mealId: 30,
        menuId: 23,
        createdAt: '2018-07-01T23:56:36.920Z',
        updatedAt: '2018-07-01T23:56:36.920Z',
      },
    },
  ],
}];

export const orders = [
  {
    id: 2,
    date: '2018-05-16',
    amount: 1500,
    userId: 1,
    mealId: 2,
    status: 'cancelled',
    createdAt: '2018-05-23T19:56:44.346Z',
    updatedAt: '2018-05-23T19:56:44.346Z',
    User: {
      id: 1,
      firstname: 'Ezinne',
      lastname: 'Njoku',
    },
    Meal: {
      id: 2,
      name: 'Rice and Beans',
      price: 1500,
    },
  },
  {
    id: 3,
    date: '2018-05-16',
    amount: 1200,
    userId: 2,
    mealId: 3,
    status: 'pending',
    createdAt: '2018-05-23T19:56:44.346Z',
    updatedAt: '2018-05-23T19:56:44.346Z',
    User: {
      id: 2,
      firstname: 'Kelechi',
      lastname: 'Njoku',
    },
    Meal: {
      id: 3,
      name: 'Egusi Semo',
      price: 1200,
    },
  },
  {
    id: 4,
    date: '2018-05-16',
    amount: 1500,
    userId: 3,
    mealId: 3,
    status: 'delivered',
    createdAt: '2018-05-23T19:56:44.346Z',
    updatedAt: '2018-05-23T19:56:44.346Z',
    User: {
      id: 3,
      firstname: 'Nnamdi',
      lastname: 'James',
    },
    Meal: {
      id: 3,
      name: 'Egusi Semo',
      price: 1200,
    },
  },
];
