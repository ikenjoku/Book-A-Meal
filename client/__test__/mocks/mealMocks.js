export const getAllMealsMock = [
  {
    name: 'test meal',
    price: 1230,
    description: 'this is a very nice meal',
    imageurl: 'https://sdfsdjhbsn/sdf.asp',
  },
];
