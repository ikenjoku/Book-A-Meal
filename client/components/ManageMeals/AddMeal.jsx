import React, { Component } from 'react';
import { connect } from "react-redux";
import { addAMeal } from "../../actions/mealActions";
import MealForm from './MealForm';

const AddMealForm = (props) => {
  console.log(props);
    return (
      <div className="manage-meals-content">
        <h3 className="center">Add a Meal</h3>
        <div className="contain-form">
          <MealForm onSubmit={(meal) => {
            props.dispatch(addAMeal(meal));
            props.history.push('/meals')
            }}
          />
        </div>
      </div>
    );
  }
export default connect()(AddMealForm);
