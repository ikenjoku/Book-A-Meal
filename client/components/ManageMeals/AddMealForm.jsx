import React, { Component } from 'react';


export default class AddMealForm extends Component {
  constructor(props) {
    super(props);
    this.state = {
      responseMessage: '',
    }
    this.handleFormSubmit = this.handleFormSubmit.bind(this);
  }
  handleFormSubmit(event){
    event.preventDefault();

    const name = event.target.elements.name.value;
    const description = event.target.elements.description.value;
    const price = event.target.elements.price.value;
    const imageurl = event.target.elements.imageurl.value;
    console.log(imageurl);
    console.log(event.target.elements.imageurl.files[0])
    console.log(new FormData(event.target.elements.imageurl.files[0]))
    const formData = {
      name,
      description,
      price,
      imageurl 
    }
    const responseMessage = this.props.handleCreateMeal(formData);
    // this.setState({ responseMessage });
    console.log(responseMessage);
    const addMealForm = document.forms.addMealForm;
    addMealForm.name.value = '';
    addMealForm.description.value = '';
    addMealForm.price.value = '';
    addMealForm.imageurl.value = '';
    
  }

  render() {
    return (
      <div>
        <h3 className="center">Add a Meal</h3>
    { this.props.responseMessage && <p className='notificationBubble'>{this.state.responseMessage}</p> }
        <div className="contain-form">
          <form 
          encType='multipart/form-data' 
          name = 'addMealForm' 
          onSubmit={this.handleFormSubmit} 
          className="add-meal-form">
            <input
              className="form-control"
              type="text"
              name="name"
              required
              placeholder="Meal Name" />

            <input
              className="form-control"
              type="text"
              name="description"
              required
              placeholder="Short Description" />

            <input
              className="form-control"
              type="number"
              name="price"
              required
              placeholder="Price" />

            <input
              className="form-control"
              type="file"
              required
              name="imageurl"/>

            <input
              className="btn-control loginsubmitBtn"
              type="submit"
              value="Add Meal" />

          </form>
        </div>
      </div>
    );
  }
}
