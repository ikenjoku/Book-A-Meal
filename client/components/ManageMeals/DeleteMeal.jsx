import React from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import { removeAMeal } from "../../actions/mealActions";

export const DeleteMeal = ({ meal, closeModal, removeAMeal }) => {

  return (
    <div className='confirm-modal'>
      <h3>Are you sure you want to delete {meal.name}?</h3>
      <div>
        <button id='confirm-delete-btn' onClick={() => {
          removeAMeal(meal.id);
          closeModal();
          }}>Yes</button>
        <button onClick={closeModal}>No</button>
      </div>
    </div>
  );
};

DeleteMeal.propTypes = {
  meal: PropTypes.object.isRequired,
  closeModal: PropTypes.func.isRequired,
  removeAMeal: PropTypes.func.isRequired,
}

export default connect(undefined, { removeAMeal })(DeleteMeal);