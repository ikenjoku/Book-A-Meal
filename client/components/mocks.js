

export const sampleOrders = {
  message: 'All orders retrieved successfully',
  orders: [
    {
      id: 2,
      date: '2018-05-16',
      amount: 1500,
      userId: 1,
      mealId: 2,
      status: 'cancelled',
      createdAt: '2018-05-23T19:56:44.346Z',
      updatedAt: '2018-05-23T19:56:44.346Z',
      User: {
        id: 1,
        firstname: 'Ezinne',
        lastname: 'Njoku',
      },
      Meal: {
        id: 2,
        name: 'Rice and Beans',
        price: 1500,
      },
    },
    {
      id: 3,
      date: '2018-05-16',
      amount: 1200,
      userId: 2,
      mealId: 3,
      status: 'pending',
      createdAt: '2018-05-23T19:56:44.346Z',
      updatedAt: '2018-05-23T19:56:44.346Z',
      User: {
        id: 2,
        firstname: 'Kelechi',
        lastname: 'Njoku',
      },
      Meal: {
        id: 3,
        name: 'Egusi Semo',
        price: 1200,
      },
    },
    {
      id: 4,
      date: '2018-05-16',
      amount: 1500,
      userId: 3,
      mealId: 3,
      status: 'delivered',
      createdAt: '2018-05-23T19:56:44.346Z',
      updatedAt: '2018-05-23T19:56:44.346Z',
      User: {
        id: 3,
        firstname: 'Nnamdi',
        lastname: 'James',
      },
      Meal: {
        id: 3,
        name: 'Egusi Semo',
        price: 1200,
      },
    },
    {
      id: 1,
      date: '2018-05-16',
      amount: 1100,
      userId: 1,
      mealId: 2,
      status: 'cancelled',
      createdAt: '2018-05-23T19:56:44.346Z',
      updatedAt: '2018-05-23T20:38:56.387Z',
      User: {
        id: 1,
        firstname: 'Ezinne',
        lastname: 'Njoku',
      },
      Meal: {
        id: 2,
        name: 'Rice and Beans',
        price: 1500,
      },
    },
    {
      id: 5,
      date: '2018-05-23',
      amount: 1500,
      userId: 2,
      mealId: 2,
      status: 'pending',
      createdAt: '2018-05-23T23:55:05.091Z',
      updatedAt: '2018-05-23T23:55:05.091Z',
      User: {
        id: 2,
        firstname: 'Kelechi',
        lastname: 'Njoku',
      },
      Meal: {
        id: 2,
        name: 'Rice and Beans',
        price: 1500,
      },
    },
    {
      id: 7,
      date: '2018-05-16',
      amount: 1200,
      userId: 2,
      mealId: 3,
      status: 'pending',
      createdAt: '2018-05-23T19:56:44.346Z',
      updatedAt: '2018-05-23T19:56:44.346Z',
      User: {
        id: 2,
        firstname: 'Kelechi',
        lastname: 'Njoku',
      },
      Meal: {
        id: 3,
        name: 'Egusi Semo',
        price: 1200,
      },
    },
    {
      id: 6,
      date: '2018-05-16',
      amount: 1500,
      userId: 3,
      mealId: 3,
      status: 'delivered',
      createdAt: '2018-05-23T19:56:44.346Z',
      updatedAt: '2018-05-23T19:56:44.346Z',
      User: {
        id: 3,
        firstname: 'Nnamdi',
        lastname: 'James',
      },
      Meal: {
        id: 3,
        name: 'Egusi Semo',
        price: 1200,
      },
    },
    {
      id: 8,
      date: '2018-05-16',
      amount: 1100,
      userId: 1,
      mealId: 2,
      status: 'cancelled',
      createdAt: '2018-05-23T19:56:44.346Z',
      updatedAt: '2018-05-23T20:38:56.387Z',
      User: {
        id: 1,
        firstname: 'Ezinne',
        lastname: 'Njoku',
      },
      Meal: {
        id: 2,
        name: 'Rice and Beans',
        price: 1500,
      },
    },
  ],
};

