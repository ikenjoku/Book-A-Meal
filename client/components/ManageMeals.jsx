import React, { Component } from 'react';

import NavBar from './NavBar.jsx'
import { mealsSample } from "./mocks";


export default class ManageMeals extends Component {
  constructor(props){
    super(props);
    this.state = {
      meals: mealsSample.meals,
    }
  }

  render(){
    return (
      <div className='contain-page'>
        <NavBar/>
        <main className="manage-meals-content">
            <h2 className="center cool-lg-text">Manage Meals</h2>
            <AddMealForm/>
            <MealList meals={this.state.meals}/>
        </main>
        <footer className="footer">
            <p>Book-A-Meal &copy;2018</p>
        </footer>
      </div>
    );
  }
}



class AddMealForm extends Component{
  constructor(props){
    super(props);
  }

  render(){
    return (
      <div>
        <h3 className="center">Add a Meal</h3>
            <div className="contain-form">
                <form className="add-meal-form">
                  <input 
                    className="form-control" 
                    type="text" 
                    name="name" 
                    placeholder="Meal Name" />
                  
                  <input 
                    className="form-control" 
                    type="text" 
                    name="descriptiion" 
                    placeholder="Short Description" />

                  <input 
                    className="form-control" 
                    type="text" 
                    name="price" 
                    placeholder="Price" />

                  <input 
                    className="form-control" 
                    type="file" 
                    name="image" 
                    placeholder="Upload an image" />
                  
                  <input 
                    className="btn-control loginsubmitBtn" 
                    type="submit" 
                    value="Add Meal" />

                </form>
            </div>
      </div>
    );
  }
}

const MealList = ({meals}) => {
  return (
    <div>
        <h1 className="meal-table-title">All Meal List</h1>
        <div id="all-meal-table">
            <div className="contain-table-headers">
                <p>Image</p>
                <p>Name</p>
                <p>Description</p>
                <p>Price</p>
                <p>Actions</p>
            </div>
            <div className="meal-table">
              { meals.map(meal =>
                <div key={meal.id} className="meal-table-item">
                    <div className="meal-table-item-img">
                      <p><img src={ require("../static/mealImg.jpg") } alt="" /></p>
                    </div>
                    <div className="meal-table-item-name">
                        <p>{meal.name}</p>
                    </div>
                    <div className="meal-table-item-desc">
                        <p>{meal.description}</p>
                    </div>
                    <div className="meal-table-item-price">
                        <p>{meal.price}</p>
                    </div>
                    <div className="meal-table-item-action">
                        <p><a className="mg-meal-btn btn-spacer">Edit</a>
                        <a className="mg-meal-btn btn-danger">Delete</a></p>
                    </div>
                </div>
              ) }
            </div>
            <hr/>
        </div>
    </div>
  );
}