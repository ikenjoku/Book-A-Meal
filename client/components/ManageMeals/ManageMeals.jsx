import React, { Component } from 'react';

import NavBar from '../NavBar/NavBar.jsx';
import AddMealForm from './AddMealForm.jsx';
import MealList from "./MealList";
// import EditMeal from './EditMeal';

import { mealsSample } from "../mocks";
import axios from 'axios';
import API from '../axiosConfig';


export default class ManageMeals extends Component {
  constructor(props) {
    super(props);
    this.state = {
      meals: [],
      message: '',
      error: '',
      selectedToUpdate: 1,
      mealToUpdate: undefined,
    }
    this.handleCreateMeal = this.handleCreateMeal.bind(this);
    this.handelUpdateMeal = this.handelUpdateMeal.bind(this);
    this.handleDeleteMeal = this.handleDeleteMeal.bind(this);
    this.clearUpdateModal = this.clearUpdateModal.bind(this);
  }

  componentDidMount(){
    this.getAllMeals();
  }

  getAllMeals(){
    API.get('/meals')
      .then(res => {
        // console.log(res);
        this.setState({ meals: res.data.meals });
      })
      .catch(err => {
        // console.log(err);
      })
  }

  clearUpdateModal(){
    this.setState({ selectedToUpdate: undefined });
  }

  handleCreateMeal(newMeal){
    const formData = new FormData(newMeal);
    formData.name = newMeal.name;
    formData.description = newMeal.description;
    formData.price = newMeal.price;
    formData.imageurl = newMeal.imageurl
    console.log(formData);
    API.post('/meals', newMeal)
      .then(({ data: { meal, message } }) => {
        // console.log(meal);
        // console.log(message);
        this.getAllMeals();
      })
      .catch(error => this.setState({ error: error.response.data.message }))
  }

  handelUpdateMeal(mealId){
    // console.log('update meal', mealId);
    const mealToUpdate =this.state.meals.filter(meal => {
      return meal.id === mealId;
    })
    this.setState({ 
      mealToUpdate: mealToUpdate[0],
      selectedToUpdate: mealId, 
    })
  }

  handleDeleteMeal(mealId){
    API.delete(`/meals/${mealId}`)
      .then(res => {
        // console.log(res);
        this.getAllMeals();
      })
      .catch(err => {
        consol.log(err);
      })
  }
  render() {
    return (
      <main className="manage-meals-content">
        <h2 className="center cool-lg-text">Manage Meals</h2>
        <AddMealForm 
          handleCreateMeal={this.handleCreateMeal} 
          statusMessage={this.state.statusMessage}
        />
        <MealList 
          meals={this.state.meals} 
          handleDeleteMeal={this.handleDeleteMeal}
          handelUpdateMeal={this.handelUpdateMeal} 
        />
        {/* <EditMeal 
          selectedToUpdate={this.state.selectedToUpdate}
          mealToUpdate={this.state.mealToUpdate}
          clearUpdateModal={this.clearUpdateModal} 
        /> */}
      </main>
    );
  }
}


