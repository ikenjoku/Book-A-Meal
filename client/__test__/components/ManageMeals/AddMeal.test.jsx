import React from 'react';
import { shallow } from 'enzyme';
import { meals } from '../../mocks';
import { AddMealForm } from '../../../components/ManageMeals/AddMealForm';

let addAMeal;
let history;
let wrapper;

describe('AddMeal Component', () => {
  beforeEach(() => {
    addAMeal = jest.fn();
    history = { push: jest.fn() };
    wrapper = shallow(<AddMealForm addAMeal={addAMeal} history={history} />);
  });

  test('should render the AddMealForm correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should handle onSubmit', () => {
    wrapper.find('MealForm').prop('onSubmit')(meals[1]);
    expect(addAMeal).toHaveBeenLastCalledWith(meals[1]);
    expect(history.push).toHaveBeenCalledWith('/meals');
  });
});
