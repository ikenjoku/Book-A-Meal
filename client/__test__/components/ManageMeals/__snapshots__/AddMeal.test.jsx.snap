// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMeal Component should render the AddMealForm correctly 1`] = `
<div
  className="manage-meals-content"
>
  <h3
    className="center"
  >
    Add a Meal
  </h3>
  <div
    className="contain-form"
  >
    <MealForm
      onSubmit={[Function]}
    />
  </div>
</div>
`;
